### DISCUSSION TITLE
Why not use "wipefs" to cleanup device, but use "shred"?

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#10273] [Q&A]
@travisn

**Comment by travisn** [comment]
Are you asking about rook's documentation or implementation? Please provide more context. But if you're cleaning up the devices directly, you can use the desired tools and rook doesn't have an opinion.

**Comment by zhucan** [comment]
About rook's implementaion. When delete ceph cluster with "cleanpolicy".

**Reply by travisn** [reply]
Did you see an issue with the implementation? Or you are just asking for background? The implementation works for scenarios we have tested though.

**Comment by zhucan** [comment]
Because of the "shred" need more time to clean, the "wpiefs" is fastly. I'm just asking for backgroud.
